### DISCUSSION TITLE
ReMAPのJapaniseが見た目上JSキー配置だが打鍵するとUKになってる。

### DISCUSSION THREAD

**Discussion by apro0666** [Yowkees/keyball#378] [General]
keyball39をセットアップしております。
組み立て上の問題を解消し、ReMAPをJapaniseに設定しキーマップを設定したところ、ReMap上はJSキー配列なのですが、テキストなどに打鍵するとUSキー配置になっております。

特にそういった記事は見当たらないのですが、keyball39上の問題かReMAPがそういう仕様なのか確認したく投稿いたしました。

JSキー配列に設定する方法をご存知であればご教示ください。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

JIS配列というのは、例えばシフトキーを押しながら；を押すと＋が入力されるキーボードということでよろしいでしょうか。
それでしたら、REMAPでJapaneseを選択すれば可能です。

おそらくPC側の設定がUS配列キーボードとなっているのではないでしょうか。

お手数ですがご確認お願いいたします。

**Reply by apro0666** [reply]
ご返信ありがとうございます。
ああなるほど、うまくいったので備忘録として記載します。

【環境】
　MAC
　Logicole MX KeysをBluetooh接続しメインキーボードとして使用
　keyballをUSB接続で設定中

MACのキーボードの種類の設定
システム設定＞キーボード＞「キーボードの種類を変更」ボタンを押下でJIS/USを設定可能。

ただしこの状態だとkeyballを操作しても、Logicole MX Keysの設定として認識される。
よって、Logicole MX KeysをBluetooh接続機器から削除した状態で、KeyballをUSB接続し
「キーボードの種類を変更」を押下、適当なボタンを押下していると、種類を判別できないとメッセージが表示された後、JIS、USの選択画面が表示されるのでここで任意の種類を設定する。
以上でkeyballがJISキーボードとして認識された。

この後、Logicole MX KeysのBluetooh接続を再設定してもKeyBallはJISとして認識された。
以上。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

解決できたようで安心しました。
また、詳細な備忘録を掲載いただきありがとうございます。
MACで同じような環境の方の助けになると思います。

以上、今後ともどうぞよろしくお願いいたします。
